src/pages/CreateRecover/CreateRecover.tsx

```diff
@@ -0,0 +1,95 @@
+import { faArrowLeft } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useNavigate } from 'react-router-dom';
+import { ModalContainer, ProgressBar } from 'components';
+import { DataTestIdsEnum } from 'localConstants';
+import { routeNames } from 'routes';
+import {
+  CreateDisclaimer,
+  CreateMnemonics,
+  CreateRecoverDownload,
+  CreateRecoverPassword,
+  CreateQuiz,
+  RecoverMnemonics
+} from './components';
+import { useCreateRecoverState } from './hooks';
+
+export const CreateRecover = () => {
+  const {
+    createdAddress,
+    keystoreString,
+    currentStep,
+    currentTitle,
+    isCreateRoute,
+    mnemonic,
+    showBackButton,
+    steps,
+    onBack,
+    onNext,
+    setCreatedAddress,
+    setKeystoreString,
+    setMnemonic
+  } = useCreateRecoverState();
+  const navigate = useNavigate();
+  const progressBarPercentageFill = Math.round((currentStep * 100) / steps);
+
+  const handleOnClose = () => {
+    navigate(routeNames.unlock);
+  };
+
+  const stepComponents = [
+    <CreateRecoverPassword
```

Comment:
maybe https://2ality.com/2017/04/conditional-literal-entries.html instead of unshift?